USAGE.md

```diff
@@ -45,19 +45,15 @@ As a *bare minimum*, you can do the following:
    ```objc
    NSString *resourcePath = [[NSBundle mainBundle] resourcePath];
    NSString *pythonHome = [NSString stringWithFormat:@"%@/python", resourcePath, nil];
-   NSString *pythonPath = [NSString stringWithFormat:@"%@/lib/python3.13", python_home, nil];
-   NSString *libDynloadPath = [NSString stringWithFormat:@"%@/lib/python3.13/lib-dynload", python_home, nil];
    NSString *appPath = [NSString stringWithFormat:@"%@/app", resourcePath, nil];
 
    setenv("PYTHONHOME", pythonHome, 1);
-   setenv("PYTHONPATH", [NSString stringWithFormat:@"%@:%@:%@", pythonpath, libDynloadPath, appPath, nil]);
+   setenv("PYTHONPATH", [NSString stringWithFormat:@"%@:%@:%@", appPath, nil]);
```

Comment:
This one definitely isn't right - there's three format specifiers, and one string substitution.